src/formatting_stack/formatters/clean_ns.clj

```diff
@@ -64,32 +64,45 @@
      (make-cleaner how-to-ns-opts refactor-nrepl-opts namespaces-that-should-never-cleaned libspec-whitelist filename)
      how-to-ns-opts)))
 
+(defmacro with-memoized-libspec-allowlist
+  "Uses refactor-nrepl's 'memoized libspec' facility for best performance,
+  while respecting formatting-stack's need of lazily requiring refactor-nrepl."
+  {:style/indent 0}
+  [& body]
+  `(do
+     (require 'refactor-nrepl.ns.libspec-allowlist)
```

Comment:
What about requiring-resolve?

Reply:
It's not available in every clojure version, so I tend not to use it in tooling projs

Reply:
F-s ships with clojure 1.10.3, so I think you're safe 😁

Reply:
Done! Also bumped r-n again for https://github.com/clojure-emacs/refactor-nrepl/blob/6529ec03c854eddcd0b62f359378d066e9d88d3d/CHANGELOG.md#332